tgui/packages/tgui-panel/styles/goon/chat-dark.scss

```diff
@@ -300,15 +300,15 @@ em {
 }
 
 .hostooc{
-  color: #323232;
+  color: #666666;
```

Comment:
yeah, this really needs early return.

Please read the contrib guideline maybe?